app/code/Magento/Reports/Test/Unit/Observer/CheckoutCartAddProductObserverTest.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Reports\Test\Unit\Observer;
+
+use Magento\Framework\Event;
+use Magento\Framework\Event\Observer;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\Quote\Model\Quote\Item as QuoteItem;
+use Magento\Reports\Model\Event as ReportsEventModel;
+use Magento\Reports\Model\ReportStatus;
+use Magento\Reports\Observer\CheckoutCartAddProductObserver;
+use Magento\Reports\Observer\EventSaver;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Unit Test for @see CheckoutCartAddProductObserver
```

Comment:
What's the point of adding @see there? :-)

Reply:
sorry, it's my old habit :) I've replaced it with a name of the class. Is that ok?